.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,59 @@
+<!--
+Hi!
+
+This is an issue tracker for conda -- the package manager.  File feature requests
+for conda here, as well as bug reports about something conda has messed up.
+
+If your issue is a bug report or feature request for:
+  * a specific conda package from Anaconda:
+      ==> file at https://github.com/ContinuumIO/anaconda-issues
+  * a specific conda package from conda-forge:
+      ==> file at the corresponding feedstock under https://github.com/conda-forge
+  * repo.continuum.io:
```

Comment:
Will your "everyday casual user" always know that `repo.continuum.io` encompasses, e.g., "conda packages from the defaults channels" (... given unchanged `defaults`, ofc...)? `conda info` or at least `conda config --show` should make that obvious, but you know, people being unobservant/in a rush/etc. 😉 

[otherwise, good idea to use a template -- structured issues FTW!]

Reply:
Top bullet is for packages from the `defaults` channel.  I meant this one to be more service problems.  I'll make that top bullet clearer.

Reply:
Haha, me talking about "unobservant" and at the same time overlooking the *very first* bullet point >.<' -- but I also like your further clarifications, thanks!

BTW: Oh and, congrats about shipping the automatic issue reporting with `4.4.x` -- by the looks of the more recent issues you opened yourself, it seems that system works quite nicely!

Reply:
The automatic reporting has been *so awesome*.  First, I think we implemented it well.  It's heavily opt-in by default; we don't ever send anything without explicit opt-in.  You actually have to type `y` or `yes`, not just hit enter (like with proceeding on `conda install`).  So it doesn't have a slimy feel.

But then, we're getting plenty of reports.  Total count for 4.4.7 is at 514 right now, and that's after we've been picking off the most noisy reports with every patch release.  It's an early warning system that probably would have made https://github.com/conda/conda/issues/6096 mostly a non-issue.  I would have known about the issue almost immediately after the release (probably in canary), and been able to pull it immediately to limit the damage.

Reply:
> heavily opt-in by default [...]  we implemented it well [...] it doesn't have a slimy feel.

Yeah, it looks like you directly hit the sweet spot, well done!

> we're getting plenty of reports

That "information flood" was what I immediately wondered about when I first saw #5531 -- but then again in the majority of cases, generated reports that directly include `conda info` and tracebacks should be much faster categorized/reproduced/fixed.

Reply:
Happy for you that it's working so well! 😄